### ISSUE TITLE
NVDA does not read Top and bottom items when I have addon Columns Review enabled

### ISSUE THREAD

**Issue by MichalSuchy** [ABuffEr/columnsReview#41]
Hello. I have NVDA 2024.4.2 and Columns Review 5.4.0 and NVDA does not automatically read me the top and bottom entries in the lists of some applications. I have to read them e.g. via the command Read Current line under the text cursor. (NVDA+Upper Arrow). Those apps are Unigram and WhatsApp and it doesn't read in open chats. Can you fix it, please? Thanks, Michal.
